packages/e2e/test/wallet/SingleAddressWallet/mint.test.ts

```diff
@@ -159,97 +105,7 @@ describe('SingleAddressWallet/mint', () => {
-    };
-
-    // Script minting policy id.
-    const policyId = Cardano.PolicyId('38299ce86f8cbef9ebeecc2e94370cb49196d60c93797fffb71d3932');
-    const scriptRedeemer: Cardano.Redeemer = {
-      // CBOR for Void redeemer.
-      data: HexBlob('d8799fff'),
-      executionUnits: {
-        memory: 13_421_562,
-        steps: 9_818_438_928
-      },
-      // Hardcoded to 0 since we only have one script
-      index: 0,
-      purpose: Cardano.RedeemerPurpose.mint
-    };
-
-    // Script data hash was precomputed with the CML (hash of datums, redeemers and language views).
-    const scriptDataHash = Hash32ByteBase16('b6eb57092c330973b23784ac39426921eebd8376343409c03f613fa1a2017126');
-
-    await walletReady(wallet);
-
-    const { collateralInput, collateralCoinValue } = await createCollateral(wallet);
-    const assetId = Cardano.AssetId(`${policyId}707572706C65`);
-    const tokens = new Map([[assetId, 1n]]);
-
-    const walletAddress = (await firstValueFrom(wallet.addresses$))[0].address;
 
-    const txProps: InitializeTxProps = {
-      collaterals: new Set([collateralInput]),
-      mint: tokens,
-      options: {
-        validityInterval: {
-          invalidBefore: undefined,
-          // eslint-disable-next-line max-len
-          invalidHereafter: undefined // HACK: setting any valid interval cause an error in the node: Uncomputable slot arithmetic; transaction's validity bounds go beyond the foreseeable end of the current era
-        }
-      },
-      outputs: new Set([
-        {
-          address: walletAddress,
-          value: {
-            assets: tokens,
-            coins: 3_000_000n
-          }
-        }
-      ]),
-      scriptIntegrityHash: scriptDataHash,
-      scripts: [alwaysFailScript],
-      witness: { redeemers: [scriptRedeemer] }
-    };
-
-    const unsignedTx = await wallet.initializeTx(txProps);
-    const finalizeProps: FinalizeTxProps = {
-      isValid: false,
-      scripts: [alwaysFailScript],
-      tx: unsignedTx,
-      witness: { redeemers: [scriptRedeemer] }
-    };
-
-    const signedTx = await wallet.finalizeTx(finalizeProps);
-
-    const [, failedTx, txFoundInHistory] = await Promise.all([
-      wallet.submitTx(signedTx),
-      firstValueFromTimed(wallet.transactions.outgoing.failed$),
-      firstValueFromTimed(
-        wallet.transactions.history$.pipe(
-          map((txs) => txs.find((tx) => tx.id === signedTx.id)),
-          filter(isNotNil),
-          take(1)
-        )
-      )
-    ]);
-
-    // Transaction should be part of the history
-    expect(txFoundInHistory).toBeTruthy();
-    // But it should also be failed since it failed the phase2 validation
-    expect(failedTx.id).toEqual(signedTx.id);
-    expect(failedTx.reason).toBe(TransactionFailure.Phase2Validation);
-    expect(txFoundInHistory.body.fee).toEqual(collateralCoinValue);
+    await burnTokens({ policySigners: [alicePolicySigner], scripts: [policyScript], wallet });
```

Comment:
Is this really necessary? If we're always minting the same AssetId, then it doesn't really 'trash' the chain that much more. This will cost a fee that could be avoided.